modules/acquia_cms_headless/tests/src/Functional/DashboardApiUrlTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Drupal\Tests\acquia_cms_headless\Functional;
+
+use Drupal\Tests\acquia_cms_headless\Traits\DashboardTableTrait;
+
+/**
+ * Tests headless dashboard API Url.
+ *
+ * @group acquia_cms
+ * @group acquia_cms_headless
+ * @group medium_risk
+ * @group push
+ */
+class DashboardApiUrlTest extends DashboardWebDriverTestBase {
+
+  use DashboardTableTrait;
```

Comment:
@deepakmishra2 Please remove this trait if your test include testing with table.